packages/ui-library/src/components/forms/toggle-switch/index.stories.ts

```diff
@@ -4,76 +4,463 @@ import { BlrToggleSwitchRenderFunction } from './renderFunction';
     },
     hintIcon: {
+      name: 'hintMessageIcon',
+      description: 'Select an icon which is displayed in front of the hint message.',
       options: [undefined, ...PureIconKeys],
       control: { type: 'select' },
       if: { arg: 'hasHint', eq: true },
-    },
-    variant: {
-      options: IconPositionVariant,
-      control: { type: 'select' },
+      table: {
+        category: 'Content / Settings',
+      },
     },
     theme: {
       options: Themes,
       control: { type: 'select' },
+      table: {
+        category: 'Appearance',
+      },
     },
-    toggleOnIcon: {
-      options: [undefined, ...PureIconKeys],
-      control: { type: 'select' },
+    showStateLabel: {
+      description: 'Choose if component has a state label.',
+      table: {
+        disable: true,
+      },
     },
-    toggleOffIcon: {
-      options: [undefined, ...PureIconKeys],
-      control: { type: 'select' },
+    label: {
+      description: 'Enter string used as label text.',
+      table: {
+        category: 'Content / Settings',
+      },
+      if: { arg: 'showStateLabel', eq: true },
+    },
+    onLabel: {
+      description: 'Enter string used as on label text.',
+      table: {
+        category: 'Content / Settings',
+      },
+      if: { arg: 'variant', eq: 'leading' },
+    },
+    offLabel: {
+      description: 'Enter string used as off label text.',
+      table: {
+        category: 'Content / Settings',
+      },
+      if: { arg: 'variant', eq: 'leading' },
+    },
+    disabled: {
+      description:
+        'Choose if component is disabled. Prevents the user to select or change the value of this component.',
+      table: {
+        category: 'States',
+      },
+    },
+    checkInputId: {
+      description: 'Unique identifier for this component.',
+      table: {
+        category: 'Technical attributes',
+      },
+    },
+    name: {
+      description: 'For a < form > element, the name attribute is used as a reference when the data is submitted.',
+      table: {
+        category: 'Technical attributes',
+      },
+    },
+    onChange: {
+      description: 'Fires when the value changes.',
+      table: {
+        category: 'Events',
+      },
+    },
+    onFocus: {
+      description: 'Fires when the component is focused.',
+      table: {
+        category: 'Events',
+      },
+    },
+    onBlur: {
+      description: 'Fires when the component lost focus.',
+      table: {
+        category: 'Events',
+      },
+    },
+    readonly: {
+      description: 'Choose if component is readonly. The user can select but not change the value of this component.',
+      table: {
+        category: 'States',
+      },
     },
   },
   parameters: {
     design: {
       type: 'figma',
       url: 'https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?node-id=3618%3A125201&mode=dev',
     },
+    layout: 'centered',
+    docs: {
+      description: {
+        component: `<Markdown>
```

Comment:
```suggestion
        component: `<markdown>
```

Reply:
Pls change to lower case since this causes an error that can only be seen wehen opening the dev tools. I had the same review suggestion before in one my of my tickets.

Reply:
I saw that. I'm not sure if that solve that problem, because now we got another warning in the dev tools.

Reply:
Yes, the other error persists but I talked too Christian about it and its still better then having both errors

Reply:
done